### PR TITLE
Enable alert prompt for fragment with content not available

### LINKED ISSUE #30: Alert user where to store members/items etc. for specific fragments, when no data available
**Is this a BUG REPORT or FEATURE REQUEST?**: 
 feature

**What happened**: Content files won't be visible except the naming structure is consistent. (i.e `content/index/my-awesome-team.md/` and `content/index/awesome-team.md/`  doesn't match)

**What you expected to happen**: To check if this is working correctly print out the resulting `{{$member}}` if it's empty then print out the location of the files in the command line `{{ printf "%s/*" $name_t }}`

### PR DESCRIPTION
**Pull request by lauragift21**
**What this PR does / why we need it**:
Add an alert prompt to fragment with content data not available. 

Optional one line note for this specific change, that can be used in a release-note or changelog.

```release-note
Add an alert prompt to fragment with content data not available. 
```

### CODE REVIEW

File: layouts/partials/fragments/member.html [hunk 1]
```diff
@@ -37,6 +38,11 @@ <h5>
           </div>
         </div>
       {{ end }}
+      {{- if eq $members 0 -}}
+        <div class="text-danger alert-danger text-center">
+          {{- printf "Currently there is no configured member content files present. To add new members add a content file within index/%s" $path -}}
```
Comment (mpourismaiel) [inline layouts/partials/fragments/member.html]: Please change the `index/%s` to something that would be correct in other pages as well.

File: layouts/partials/fragments/items.html [hunk 2]
```diff
@@ -38,6 +39,11 @@ <h5>
         </div>
       {{- end }}
       <div class="row justify-content-center align-items-start">
+        {{- if eq $items 0 -}}
+        <div class="text-danger alert-danger">
+            {{- printf "Currently there is no configured items content files present. To add new items add a content file within index/%s" $path -}}
```
Comment (mpourismaiel) [inline layouts/partials/fragments/items.html]: Change the index here as well.

File: layouts/partials/fragments/items.html [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {{- $items := .resources.Match (printf "%s/*" .Name) -}}
+{{- $path := .Name -}}
```
Comment (mpourismaiel) [inline layouts/partials/fragments/items.html]: Isn't there any way to avoid doing this? It's a bit annoying.

Comment (lauragift21) [reply layouts/partials/fragments/items.html]: hehe yeah i didn't want to bring that back seeing it was already removed but i needed a way to get the filename

**Pull request merged**